solvespace: New Package

Kreijstal opened the pull request:
the 1487.patch refers to the PR, and because this patch is on release it will not exactly work for the tagged version, so the 002*.patch is a hotfix which should be removed whenever they tag a new version.

lazka commented on mingw-w64-solvespace/PKGBUILD:
```diff
@@ -0,0 +1,66 @@
+
+prepare() {
+  cd "${srcdir}/${_realname}-${pkgver}"
+  patch -N -p1 -i "${srcdir}/1487.patch" || true
+  patch -N -p1 -i "${srcdir}/002-not-in-msys2-temp.patch" || true
```
why the "|| true" ?

Kreijstal replied:
the first patch is from PR but it's from upstream which is not the version we are packaging, the rejects are then fixed/backported in 002-not-in-msys2-temp.patch, however the second one a || true is not necessary, you're right.

lazka replied:
I'd argue it doesn't make sense in both cases. It shouldn't fail, and if it does we want to know.

Kreijstal replied:
I guess I can manually remove the reject chunks

lazka commented:
the resulting package is empty, so something is broken in the package() phase

Kreijstal commented:
> the resulting package is empty, so something is broken in the package() phase

It seems it's an upstream problem..

Kreijstal committed:
[new package] solvespace: add version 3.2

Kreijstal committed:
solvespace: fix clangarm64 build by using system OpenGL instead of in-tree ANGLE

The in-tree ANGLE (OpenGL ES → DirectX translation) does not compile
on ARM64 Windows. On clangarm64, use OPENGL=1 which falls back to
find_package(OpenGL) using Windows' system OpenGL32.lib.
